src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -79,13 +80,12 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
   state_ = State::kStarting;
 
   auto capture_result = thread_pool->Schedule(
-      [this, process_id = process.pid(), &module_manager,
-       selected_functions = std::move(selected_functions), selected_tracepoints,
-       frame_track_function_ids = std::move(frame_track_function_ids), collect_thread_state,
-       samples_per_second, unwinding_method, enable_introspection,
+      [this, &process, &module_manager, selected_functions = std::move(selected_functions),
+       selected_tracepoints, frame_track_function_ids = std::move(frame_track_function_ids),
+       collect_thread_state, samples_per_second, unwinding_method, enable_introspection,
        max_local_marker_depth_per_command_buffer, collect_memory_info,
        memory_sampling_period_ns]() mutable {
-        return CaptureSync(process_id, module_manager, selected_functions, selected_tracepoints,
+        return CaptureSync(process, module_manager, selected_functions, selected_tracepoints,
```

Comment:
I would say pass api_table_infos here instead of process.. and generate it before this call, the way we do it for everything else here.

Reply:
Reverted to original signature.